03_letters/nicholeL/main_test.go

```diff
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	excepted := strconv.Quote("a:2\nb:1\n")
+	actual := strconv.Quote(buf.String())
+	if excepted != actual {
+		t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", excepted, actual)
+	}
+
+}
+
+func TestLetters(t *testing.T) {
+	testCases := map[string]struct {
+		input    string
+		excepted map[rune]int
+	}{
+		"one":   {"aba", map[rune]int{97: 2, 98: 1}},
```

Comment:
You can declare another variable inside the struct called `testName` or something similar, and the testName can be more descriptive than `one`, `two`, `three`.

Reply:
thanks for your advice